modules/EventTracker.js

```diff
@@ -0,0 +1,187 @@
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(['underscore', 'backbone', './ServiceCell'], factory);
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('underscore'), require('backbone'), require('./ServiceCell'));
+  } else {
+    root.Torso = root.Torso || {};
+    root.Torso.EventTracker = factory(root._, root.Backbone, root.Torso.ServiceCell);
+  }
+}(this, function(_, Backbone, ServiceCell) {
+  'use strict';
+
+  var EventTracker = ServiceCell.extend({ 
```

Comment:
Triggering all events through a single "Events" channel would create a Pub-Sub-like system. The hard part about that is that I'd want to, in many situations, subscribe to events that both match an event and the instance that triggered it. But, it allows for ease of access to all triggered events and a centralized place to pick up all events of a certain type.

If we trigger on Torso.Events, then the instance firing the event needs to be included in the event.

I'm thinking of the use-case of wanting to attach my own "pre-event-dom-callback" and "post-event-dom-callback"s in a view. This should be able to be performed inside the view without the help of Torso.Events. Perhaps routing all actual events through Torso.Events, and separately invoking (default noop) methods in the view before and after.

Is it possible to create something like: this.listenTo(Torso.Events, 'my-event', instanceWhoCalledMyEvent)? Perhaps: Torso.Events.notify(this, 'my-event', instanceWhoCalledMyEvent)?